rdkit/Chem/MolStandardize/charge.py

```diff
@@ -36,23 +34,23 @@ def __init__(self, name, acid, base):
         :param string acid: SMARTS pattern for the protonated acid.
         :param string base: SMARTS pattern for the conjugate ionized base.
         """
-        log.debug('Initializing AcidBasePair: %s', name)
+        log.debug(f'Initializing AcidBasePair: {name}')
         self.name = name
         self.acid_str = acid
         self.base_str = base
 
     @memoized_property
     def acid(self):
-        log.debug('Loading AcidBasePair acid: %s', self.name)
+        log.debug(f'Loading AcidBasePair acid: {self.name}')
         return Chem.MolFromSmarts(self.acid_str)
 
     @memoized_property
     def base(self):
-        log.debug('Loading AcidBasePair base: %s', self.name)
+        log.debug(f'Loading AcidBasePair base: {self.name}')
         return Chem.MolFromSmarts(self.base_str)
 
     def __repr__(self):
-        return 'AcidBasePair({!r}, {!r}, {!r})'.format(self.name, self.acid_str, self.base_str)
+        return fr'AcidBasePair({self.name}, {self.acid_str}, {self.base_str})'
```

Comment:
This is not identical. Use this:
```suggestion
        return f'AcidBasePair({self.name!r}, {self.acid_str!r}, {self.base_str!r})'
```

Reply:
```
>>> fr'AcidBasePair({self.name}, {self.acid_str}, {self.base_str})'
'AcidBasePair(abc, def, xyz)'
>>> f'AcidBasePair({self.name!r}, {self.acid_str!r}, {self.base_str!r})'
"AcidBasePair('abc', 'def', 'xyz')"
```